### ISSUE TITLE
PB 2.0 on Firefox 50.0.2 (Linux) breaks IRCcloud.com

### ISSUE THREAD

**Issue by downeym** [EFForg/privacybadger#1055]
As of the latest (newest) version of Privacy Badger (2.0), running on Firefox 50.0.2 (standard distributed with Fedora 25) causes IRCcloud.com to never get past its loading state after signing in. 

Disabling the Add-On allows the site to load fine.

Interestingly, there are 0 items blocked, and even with the add-on "disabled for this site" (IRC Cloud) the page still doesn't load.

**Comment by downeym** [comment]
Oh, and I did try the uninstall-and-reinstall suggestion from other issues and Twitter, and sadly that didn't fix the problem.

**Comment by spinda** [comment]
Experiencing the same issue; reinstalling doesn't fix it for me either.

https://gyazo.com/1f75c19d0265b06b9292fe5a2ce62966

Firefox Developer Edition 52.0a2 (2016-12-13)

**Comment by downeym** [comment]
Reproduced same problem on Firefox 50.1.0 on OS X 10.12.1.

**Comment by groovecoder** [comment]
Looks like the same duplicate `ERROR.captureStackTrace` issue causing https://github.com/EFForg/privacybadger/issues/1050 and fixed in https://github.com/EFForg/privacybadger/pull/1053. Suggest closing this as duplicate and merging https://github.com/EFForg/privacybadger/pull/1053 ASAP.

**Comment by cooperq** [comment]
This should be fixed by version 2016.12.15.1 please re-open if not.

**Comment by tigt** [comment]
I just had this issue, after FF Nightly crashed (as it's wont to do). I previously had www.irccloud.com whitelisted and things worked okay.

Firefox OSX 54.0a1 (2017-01-24) (64-bit)
Privacy Badger 2017.1.24.1

**Comment by tigt** [comment]
CAUGHT ONE. I have the output of the JS and browser consoles at https://gist.github.com/tigt/c97b56d780864cc01d1a1a6d70e2ef15

**Comment by pimeys** [comment]
Having the same issue here. I can see the JS error:
```
TypeError: ERROR.captureStackTrace is not a function[Learn More]
```

Firefox 50.0.1 + PB 2017.1.24.1

**Comment by spinda** [comment]
I'm also experiencing this, on Firefox Developer Edition 52.0a2 (2017-01-20) (64-bit) (Windows 10), with Privacy Badger 2017.1.24.1. Could this issue be reopened? (cc @cooperq)

**Comment by ghostwords** [comment]
This should be fixed again with Privacy Badger version 2017.1.26.1, released a few minutes ago.
